### DISCUSSION TITLE
Welcome to Orange OS LE CLI Installer Discussions!

### DISCUSSION THREAD

**Discussion by michaeleldar** [Orange-OS-LE/CLI_Installer#1] [Announcements]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by UniqueName12345** [comment]
I am DifferentDance8. I think the rest is self explanatory from the work I've done over on my fork.
